### ISSUE TITLE
jQuery minify issue with new hugo v0.113.0+extended

### ISSUE THREAD

**Issue by tfsojon** [gohugoio/hugo#11132]
### This issue with the latest release.

- Using Hugo (`v0.113.0+extended`)
- [jquery-3.7.0.js](https://code.jquery.com/jquery-3.7.0.js)
```
{{ $scripts := slice }}
{{ range site.Params.plugins.js}}
{{ $scripts = $scripts | append (resources.Get .link) }}
{{ end }}
{{ $scripts = $scripts | append (resources.Get "js/script.js") }}
{{ $scripts = $scripts | resources.Concat "/js/script.js" | minify | fingerprint "sha512" }}
```

![image](https://github.com/gohugoio/hugo/assets/87603786/e347df59-8964-4569-92d0-47186c99f1e6)

### without minify its working

```
{{ $scripts := slice }}
{{ range site.Params.plugins.js}}
{{ $scripts = $scripts | append (resources.Get .link) }}
{{ end }}
{{ $scripts = $scripts | append (resources.Get "js/script.js") }}
{{ $scripts = $scripts | resources.Concat "/js/script.js" | fingerprint "sha512" }}
```

**Comment by bep** [comment]
Do you any info about the last version of Hugo this was OK?

**Comment by bep** [comment]
Yes, this is a bug and needs to be fixed, but I think you would get a better JS setup if you rewrote the above to use 

* https://gohugo.io/hugo-pipes/js/
* which supports imports directly from `assets` (e.g. your jquery)
* has an option for minification

**Comment by bep** [comment]
I just quickly tested this with that jQuery version and even minified it worked OK (I just ran some basic jquery examples found onklin) ... Which I guess it means that there's a certain part of jQuery that you use that somehow gets mingled. Which means that the upstream author probably needs some more info.

**Comment by tfsojon** [comment]
I'm setting up a theme and will link the repo to this issue.

**Comment by tfsojon** [comment]
@bep https://github.com/tfsojon/jquery-hugo

**Comment by bep** [comment]
Not that it makes any difference in this case (it fails also starting with the unminified jQuery source), but doing minification on already minified JS sources is not something I would recommend.

**Comment by jmooring** [comment]
@tfsojon How do I reproduce the problem with the repo you linked to?

**Comment by jmooring** [comment]
While reducing this to a simple example, I found that the problem is exposed when the minified resource is rendered inline ([see original code](https://github.com/tfsojon/jquery-hugo/blob/main/themes/jquery-hugo/layouts/partials/script.html#L30)).

For example, this fails:

```html
<script>{{ $js.Content | safeJS }}</script>
```

This does not fail:

```html
<script src="{{ $js.RelPermalink }}"></script>
```

---

Minimal example:

```text
git clone --single-branch -b hugo-github-issue-11132 https://github.com/jmooring/hugo-testing hugo-github-issue-11132
cd hugo-github-issue-11132
```

Open the browser's console, then:

```text
hugo server
```

**Comment by jmooring** [comment]
I can confirm that was fixed with [tdewolff/minify v2.12.7](https://github.com/tdewolff/minify/releases/tag/v2.12.7). Verified against @tfsojon's test repo, as well as the minimal example above. Tested with minified script inline and external.

**Comment by tfsojon** [comment]
Thank You. Issue Solved with [v0.114.0](https://github.com/gohugoio/hugo/releases/tag/v0.114.0)

**Comment by bep** [comment]
v0.114.0 was a ... workaround.
